app/Http/Controllers/Intake/IntakeController.php

```diff
@@ -3,12 +3,35 @@
 namespace App\Http\Controllers\Intake;
 
 use App\Http\Controllers\Controller;
+use App\Services\PdfIntakeFormService;
+use App\Services\NeonApiService;
+use App\Services\NeonDataTransformer;
+use Illuminate\Support\Facades\Storage;
 use Inertia\Inertia;
 
 class IntakeController extends Controller
 {
-    public function index()
+    // public function index()
+    // {
+    //     return Inertia::render('Intake/Index');
+    // }
+
+    public function generatePdf(int $participantId, NeonApiService $neonApi, NeonDataTransformer $transformer, PdfIntakeFormService $pdfService)
```

Comment:
Move this `generatePdf` method to a service class: `app/Services/PdfService.php`